Cyra/part_1/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(hello_world LANGUAGES CXX)
+add_library(one STATIC src/test.cpp include/random.h)
+project(part_1)
```

Comment:
why do you have a second project call here? there should only ever be one project() call in a single cmake project